changelog.yaml

```diff
@@ -59,6 +59,12 @@
         This change is fixing the heartbeat.
       pullRequestNumber: 275
       jiraIssueNumber: 1131
+    - type: security
+      impact: patch
+      title: update go version
+      description: |-
+        Update go version to [1.16.9](https://golang.org/doc/go1.16) to fix [CVE-2021-38297](https://nvd.nist.gov/vuln/detail/CVE-2021-38297)
```

Comment:
Add warning to update also local dev environment?
What about the go version used in the image used in our pipeline?

Reply:
There will be a message printed if you use build.sh with the wrong go version.
I'm not sure if we should put depeloper information to the release notes.

Reply:
```suggestion
        Update go version to [1.16.10](https://golang.org/doc/go1.16) to fix [CVE-2021-38297](https://nvd.nist.gov/vuln/detail/CVE-2021-38297)
```